cdc/sink/mysql.go

```diff
@@ -347,6 +361,26 @@ func (s *mysqlSink) concurrentExec(ctx context.Context, rowGroups map[string][]*
 	return eg.Wait()
 }
 
+func txnRowLimiter(rows []*model.RowChangedEvent, maxTxnRow int) [][]*model.RowChangedEvent {
+	evaluateResultLength := (len(rows) / maxTxnRow) + 1
+	result := make([][]*model.RowChangedEvent, 0, evaluateResultLength)
```

Comment:
Can we only maintain a slice, which holds the index of RowChangedEvent we will consume each time, such as 10 events, batch size is 4, we get a slice of [4, 8, 10] and we consume rows[:4], rows[4:8], rows[8:10]. Then we save one time to copy some pointers.